custom/icds_reports/templates/icds_reports/base_minimal.html

```diff
@@ -18,6 +18,9 @@
     <script src="{% statici18n LANGUAGE_CODE %}"></script>
     <script src="{% static 'hqwebapp/js/initial_page_data.js' %}"></script>
     {% block head %}{% endblock head %}
+    {% if less_debug %}
+      <script src="{% static 'icds_reports/mobile/css/less.min.js' %}" ></script>
```

Comment:
this still uses its own `less.js` file because HQ's version seems to mess up a few parts of the styles (guessing a version compatibility issue?).

this doesn't seem to be a problem if I use offline generation, although that might be a problem with my local setup.

any thoughts?

Reply:
I don't know specifically what would be the problem, but version incompatibility sounds plausible, I didn't realize how old HQ's version is. I think it would be fine to update less in `bower.json` and then in `hqwebapp/base.html`, given that it's only included when `less_debug` is true.

Reply:
that makes sense to me. do you have a sense of how the `less.js` version relates to the version that happens offline? (I can probably google and look in commcare cloud somewhere for this, just wondering if you knew off the top of your head - trying to figure out if there is an expected pinning that happens or whether it's a free for all between them. [our dev setup doesn't specify a version for offline](https://github.com/dimagi/commcare-hq/blob/master/DEV_SETUP.md#option-3-compress-offline-just-like-production)).

Reply:
Our servers use 2.7.2, I don't think we make any attempt to keep that version in sync with less.js (though now that I say that, we probably should).

Reply:
https://github.com/dimagi/commcare-hq/pull/26212